src/Apps/W1/EDocumentConnectors/Continia/App/src/Onboarding/ContiniaParticipation.Table.al

```diff
@@ -0,0 +1,290 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+namespace Microsoft.EServices.EDocumentConnector.Continia;
+
+using System.Email;
```

Comment:
Could you look at the problems in vscode and use the action associated with the usings not sorted to sort all files in the project?
We'd like to turn that codecop back on :)